docs/3.0rc/get-started/index.mdx

```diff
@@ -1,36 +1,74 @@
 ---
 title: Welcome to Prefect
-description: Get started with Prefect, the easiest way to orchestrate and observe your data pipelines
-# TODO Figure out how to do this: hide_table_of_contents: true
+description: Prefect is a workflow orchestration tool that empowers developers to build, observe, and react to data pipelines.
+mode: wide
 ---
 
-Prefect is a workflow orchestration tool empowering developers to build, observe, and react to data pipelines.
+<a href="https://app.prefect.cloud" class="bg-primary-dark dark:bg-primary-light rounded-md px-4 mr-2 inline-flex text-sm text-white border border-primary-dark dark:border-primary-light">
```

Comment:
Since these actions are already available in once click, I think we can remove these buttons and pull the rest of the great content you've written up the page.